e2e/session-launcher.test.ts

```diff
@@ -1,23 +1,187 @@
 import {
-  createSession,
   deleteSession,
   loginAsAdmin,
   loginAsUser,
   navigateTo,
 } from './test-util';
-import { test, expect } from '@playwright/test';
+import { test, expect, Page } from '@playwright/test';
+
+const getMenuItem = (page: Page, menuName: string) => {
+  return page.getByRole('link', { name: menuName });
+};
+
+const getCardItemByCardTitle = (page: Page, title: string) => {
+  return page.locator(`.ant-card:has-text("${title}")`);
+};
+
+const getStartSessionButton = (page: Page) => {
+  return page.getByLabel('power_settings_new');
+};
+
+const createInteractiveSessionOnSessionStartPage = async (
+  page: Page,
+  sessionName: string,
+) => {
+  const batchRadioButton = page
+    .locator('label')
+    .filter({ hasText: 'Interactive' })
+    .locator('input[type="radio"]');
+  await batchRadioButton.check();
+  const sessionNameInput = page.locator('#sessionName');
+  await sessionNameInput.fill(sessionName);
+  await page.getByRole('button', { name: 'Skip to review' }).click();
+
+  await page.locator('button').filter({ hasText: 'Launch' }).click();
+  await expect(page.locator('.ant-modal-confirm-title')).toHaveText(
+    'No storage folder is mounted',
+  );
+  await page.getByRole('button', { name: 'Start' }).click();
+};
+
+const createBatchSessionOnSessionStartPage = async (
+  page: Page,
+  sessionName: string,
+) => {
+  const batchRadioButton = page
+    .locator('label')
+    .filter({ hasText: 'Batch' })
+    .locator('input[type="radio"]');
+  await batchRadioButton.check();
+  const sessionNameInput = page.locator('#sessionName');
+  await sessionNameInput.fill(sessionName);
+  const BatchModeConfigurationCard = page.locator(
+    'div.ant-card:has-text("Batch Mode Configuration")',
+  );
+  expect(BatchModeConfigurationCard).toBeVisible();
+  const startupCommand =
+    BatchModeConfigurationCard.getByLabel('Startup Command');
+  expect(startupCommand).toBeVisible();
+  await startupCommand.fill('sleep 30');
```

Comment:
The above expect statement will be automatically validated in .fill('sleep 30'). It looks like unnecessary variable declarations and expect are used.